uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DroidKaigiApp.kt

```diff
@@ -17,38 +17,36 @@ import io.github.droidkaigi.feeder.core.use
 
 @Composable
 fun DroidKaigiApp(firstSplashScreenState: SplashState = SplashState.Shown) {
-    ProvideAppViewModel(viewModel = appViewModel()) {
```

Comment:
Remove unnecessary provider nesting.

Reply:
I didn't notice it! Do you have eagle eyes?  🦅  👍